wee_slack.py

```diff
@@ -1108,7 +1108,7 @@ def connect(self):
                 # The fast reconnect failed, so start over-ish
                 for chan in self.channels:
                     self.channels[chan].got_history = False
-                s = SlackRequest(self.token, 'rtm.start', {}, retries=999)
+                s = SlackRequest(self.token, 'rtm.start', {"batch_presence_aware": True}, retries=999)
```

Comment:
`urllib.urlencode` will turn this into `batch_presence_aware=True`, not `batch_presence_aware=true`, so you won't get batched presences.

Though, I tried sending `"true"` instead and still doesn't get batched presences (i.e. `presence_change` has a `user` field instead of a `users` field). When sending `1` instead, I get them, so that seems like a bug in the docs or API. We should send `1` so we actually get the batched events.

I'll report a bug to slack on this.

Reply:
✓